hack/common.sh

```diff
@@ -1001,6 +1001,23 @@ function os::build::find-binary() {
 }
 readonly -f os::build::find-binary
 
+# os::build::get-go-bin-location-from-gopath locates a go install binary in GOPATH directories 
+# and returns the first path where the binary was found. 
+function os::build::get-go-bin-location-from-gopath() {
```

Comment:
Since you're taking a parameter, it's best practice to internalize it right away:
```sh
local binary_name="$1"
```